verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -373,7 +435,7 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
                                 onClick={(): void => this.scrollTo('symptoms')}
                             >
                                 {this.tableOfContentsIcon({
-                                    isChecked: values.symptoms.length > 0,
+                                    isChecked: values.symptoms?.length > 0,
```

Comment:
Does this statement have problems when symptoms is null? Or is there some ts craziness where `null > 0` is a valid check
If there are problems it could be `(values.symptoms?.length ?? 0) > 0`

Reply:
This was throwing a "can't read length property of undefined" when passing in an existing case without symptoms so I added the necessary ? safeguard. I don't expect any issue with null symptoms. Added a unit test to check that.